docs/rules/prefer-starts-ends-with.md

```diff
@@ -0,0 +1,18 @@
+# Prefer `.startsWith` and `.endsWith` 
```

Comment:
> Prefer `String#startsWidth` & `String#endsWidth` over a regex

@jfmengels @SamVerschueren ⬆️ Any better suggestions?

Use this in the readme description too.